### ISSUE TITLE
Chat Highlight issue

### ISSUE THREAD

**Issue by Pacmanarea51** [night/betterttv#5919]
Hi,

I have encountered an issue where the BTTV highlight (and pin highlights) option highlights messages I sent cointaining a highlighted keyword. I can't find an option to filter own messages. 

This hasn't happened until a week or so ago, before no issues. Also, it pins my own message to the top of chat twice; one instantly as I send the message and another one half a second or so later. 

I imported BTTV settings from file as my storage device had to be changed and aftert that the issue has existed. I have tried resetting the settings and adding the highlighted keywords again but no difference.

**Comment by night** [comment]
You can update your highlighted keywords in the BetterTTV settings. You have mistakenly added yourself as a "Username" highlight. Remove that to avoid highlighting yourself.

**Comment by Pacmanarea51** [comment]
This is not fixing the issue. I have keywords like *song* or *music* enabled so I can answer people a currently playing song in a stream. I can't delete that without losing highlighted messages from other users which is the reason I added the highlight in the first place. 

Also, I don't want to delete my username as a highlighted keyword as some people do not use "@" symbol to mention me. 

Again this hasn't been an issue since a couple weeks ago (I noticed the settings page had changed and there were new options on the pinned highlight settings.)
